pytermgui/ansi_interface.py

```diff
@@ -75,7 +76,8 @@
 
 
 class Color:
-    """Class to store various color utilities
+    """
```

Comment:
I've gone to great detail in your previous PR about these changes not being necessary.

Reply:
Just for readability

Reply:
I remember it but when docstring is multiline it is better to make it like this. (Even other projects do like this for the sake of readability)

Reply:
Read [the Google styleguide](https://google.github.io/styleguide/pyguide.html#382-modules). This project loosely follows this, in that we use the ending `"""` without a linebreak.

Reply:
Alright